apps/web/src/pages/templates/components/templateToFormMappers.ts

```diff
@@ -49,8 +49,6 @@ const mapDigestStep = (item: IStepEntity): IStepEntity => ({
   ...item,
   metadata: {
     ...item.metadata,
-    type: DigestTypeEnum.REGULAR,
```

Comment:
This will break old backoff steps

Reply:
can you say more about why? the previous code was also breaking things... after you saved the scheduled digest and did a page refresh it erased that information and set it to regular with backoff, which is not true

Reply:
Because if we do not add the backoff: true if it has type backoff since before it will not open backoff panel as it should...

Reply:
so we have to do that by checking the "old" backoff type